components/scream/src/share/atm_process/atmosphere_process.hpp

```diff
@@ -483,6 +498,19 @@ class AtmosphereProcess : public ekat::enable_shared_from_this<AtmosphereProcess
   std::list<std::pair<CheckFailHandling,prop_check_ptr>> m_precondition_checks;
   std::list<std::pair<CheckFailHandling,prop_check_ptr>> m_postcondition_checks;
 
+  // Column local mass and energy conservation check
+  std::pair<CheckFailHandling,prop_check_ptr> m_column_conservation_check;
+
+  // Store data related to this processes conservation check.
+  struct ColumnConservationCheckData {
+    // Boolean which dictates whether or not this process
+    // contains the mass and energy conservation checks.
+    bool has_check;
+    // Tolerance used for the conservation check
+    Real tolerance;
```

Comment:
Do you think it makes sense to use the same tolerance for all conservation checks? Even for the MassEnergy check itself, is it ok to use the same tolerance for mass and energy?

Reply:
I'm OK with this if EAM uses just one tol. In principle a process should conserve mass and energy to nearly machine precision, so we ought to be able to set a tolerance of 1e-15 or 1e-14 eventually.

Reply:
EAM hard codes its tolerances. 1e-10 for mass, 1e-14 for energy.

Reply:
I'll separate mass and energy into their own tolerance, and make them global parameters.

Reply:
Yeah, I like that idea. I see some inconsistencies with formatting yaml section names: `Time Stepping` and `Scorpio` (spaces, capital letters) vs. `initial_conditions`, `atmosphere_processes`, etc. Should I add tolerances to Debug, and then in a separate PR make the section names consistent?